### ISSUE TITLE
Questions about hyper-parameters to balance the two DVH loss terms

### ISSUE THREAD

**Issue by ChengSongyang** [ukaukaaaa/BeamDosePrediction#1]
Hi, I notice one of your novel contribution is the DVH Loss, but the training code is not released. So could you please release the remaining code? How the hyper-parameters (α and β) are set when balance the two DVH loss terms(LvDVH and LcDVH)?

**Comment by ukaukaaaa** [comment]
> Hi, I notice one of your novel contribution is the DVH Loss, but the training code is not released. So could you please release the remaining code? How the hyper-parameters (α and β) are set when balance the two DVH loss terms(LvDVH and LcDVH)?

Hi, thanks for your comment.

For the data-preprocessing part and training part code, you can refer to [https://github.com/LSL000UD/RTDosePrediction/tree/main/RTDosePrediction/Src/C3D](url)
Only data preprocessing and training framework should be enough.

About the hyper-parameters, for your information, we set α and β as 0.5.
